### ISSUE TITLE
Merge braille utils and dotify api

### ISSUE THREAD

**Issue by joeha480** [brailleapps/issues#12]
The braille utils api and dotify api should be merged into the same api for the following reasons:
- reduce maintanence overhead
- improve symmetry and synergy
- increase focus on the apis as a whole

Note that the task-api should **not** be included in the merge.

**Comment by joeha480** [comment]
Depends on #9

**Comment by joeha480** [comment]
Duplicates brailleapps/braille-utils.api#3
